assets/js/components/KeyMetrics/KeyMetricsCTAFooter.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * KeyMetricsCTAFooter component.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Cell, Row } from '../../material-components';
+import Link from '../Link';
+
+export default function KeyMetricsCTAFooter( { onActionClick = () => {} } ) {
+	return (
+		<Row>
+			<Cell size={ 12 }>
+				<Link onClick={ onActionClick }>
+					{ __( 'Maybe later', 'google-site-kit' ) }
```

Comment:
The style doesn't quite match the design. This should use `surfaces/on-background-variant` but it's currently green.

This applies to the setup CTA too, so fixing it here should cover that one too 👍

Reply:
Fixed the colour including a few more styles, thank you!